### PR TITLE
[DEVELOPER-4845 and DEVELOPER-4844] Image and video heroes

### PR DESCRIPTION
**Pull request by kerrivader**

### CODE REVIEW

**Comment by paulrobinson** [comment]
this is ok to test

**Comment by LightGuard** [comment]
@kerrivader could you please clean up the merge conflict?

**Comment by kerrivader** [comment]
@LightGuard Rebased

**Comment by KamiQuasi** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

File: _docker/drupal/drupal-filesystem/web/config/sync/system.theme.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 admin: seven
-default: rhd
+default: rhdp
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/config/sync/system.theme.yml]: This should go back to rhd.

Comment (kerrivader) [reply _docker/drupal/drupal-filesystem/web/config/sync/system.theme.yml]: fixed

**Pull request merged**